Hủaıbaı/assets/minecraft/lang/toa_tqg.json

```diff
@@ -118,8 +118,32 @@
   "disconnect.genericReason": "%s",
   "disconnect.quitting": "Chủfaq shảı lủaq",
   "options.language": "Zủ...",
+  "options.sounds": "Gỉaq ru Lảq...",
+  "options.sounds.title": "Pỏ Gỉaq ru Lảq ga Kẻajuao",
   "record.nowPlaying": "Nỉ gỉaq ku: %s",
   "options.languageWarning": "Dảı bû mủo dủe sé tủa nỏ zủ zay",
+  "options.online": "Zỉqrıaq...",
+  "options.online.title": "Pỏ Zỉqrıaq ga Kẻajuao",
+  "options.difficulty": "Nỏacaı",
```

Comment:
'Cảı' means 'strong' rather than 'strength', so I'd suggest 'Nẻ nỏa' for this purpose